lib/pbench/server/api/resources/datasets_metadata.py

```diff
@@ -80,7 +80,7 @@ def _get(self, params: ApiParams, request: Request) -> Response:
             params: API parameters
             request: The original Request object containing query parameters
 
-        GET /api/v1/datasets/metadata?name=dname&metadata=dashboard.seen,server.deletion
+        GET /api/v1/datasets/metadata?name=dname&metadata=global.seen,server.deletion
```

Comment:
Given the comment [here](https://github.com/distributed-system-analysis/pbench/blob/ea1cc24f0ed2ac4c119702fb9ae22e846984b1d7/docs/API/metadata.md?plain=1#L42), this should presumably be `global.dashboard.seen`.  Ditto lines 104 and 118.